src/mixer/playermanager.cpp

```diff
@@ -608,6 +608,13 @@ TrackPointer PlayerManager::getLastEjectedTrack() const {
     return nullptr;
 }
 
+TrackPointer PlayerManager::getSecondLastEjectedTrack() const {
+    if (m_pLibrary) {
+        return m_pLibrary->trackCollectionManager()->getTrackById(m_secondLastEjectedTrackId);
+    }
+    return nullptr;
```

Comment:
I'm unfamiliar with `PlayerManager`. Is it possible for `m_pLibrary` to be `nullptr`?

```suggestion
    VERIFY_OR_DEBUG_ASSERT(m_pLibrary != nullptr) {
        return nullptr; 
    }
    return m_pLibrary->trackCollectionManager()->getTrackById(m_secondLastEjectedTrackId);
```

Reply:
:+1: 
`m_pLibrary` should be set during CoreServices' init.